### DISCUSSION TITLE
Tesseract scene without geometry

### DISCUSSION THREAD

**Discussion by johnwason** [tesseract-robotics/tesseract#597] [Q&A]
I have an application where the robot and scene kinematics are known, but the geometry may not be. Is it possible to build the scene with just kinematic information and run the planners, or will they fail due to the lack of geometry?

**Comment by Levi-Armstrong** [comment]
Are you constructing it from a URDF?

**Comment by johnwason** [comment]
No, in this case it would be constructed using the API.

**Comment by johnwason** [comment]
I guess I can add some cylinders as stand-ins for the joints
